seq2science/rules/quantification.smk

```diff
@@ -280,9 +280,9 @@ elif config["quantifier"] == "kallistobus":
             Align reads against a transcriptome (index) with kallistobus and output a quantification file per sample.
             """
             input:
-                barcodefile=config["barcodefile"],
                 basedir=get_kb_dir,
-                reads=rules.fastq_pair.output.reads
+                reads=rules.fastq_pair.output.reads,
+                barcodefile=expand(["{barcodefile}"] if 'barcodefile' in config else [],**config),
```

Comment:
it's nice that you have this in the input fields, as it's clear in the DAG this file is used.

not sure if this works, but couldn't the file path simply be `config.get("barcodefile", []),`?

Reply:
yes, that one gives equivalent results but is quite a bit shorter 👍